via/app.py

```diff
@@ -29,6 +29,7 @@
     "enable_front_page": {"formatter": asbool},
     "youtube_transcripts": {"formatter": asbool},
     "api_jwt_secret": {"required": True},
+    "youtube_api_key": {"required": True},
```

Comment:
Making this envvar required. I don't think we want to have to worry about two different behaviours/configurations of Via: with and without a YouTube API key. We can just set the envvar in every deployment that we have

Reply:
If we are going to add a lot of integrations to Via over time then I am concerned about stacking up the number of API keys etc. that have to be configured before you can run it. I think it would be fine to make this API key a prerequisite of using the video player app though, as you can still use it for proxying PDFs etc. without that.